### PR TITLE
Add kudo plugin

### PR DESCRIPTION
**Pull request by jbarrick-mesosphere**

### CODE REVIEW

File: plugins/kudo.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kudo
+spec:
+  version: "v0.4.0"
+
+  shortDescription: Declaratively build, install, and run operators using KUDO.
+  homepage: https://kudo.dev/
+  description: |
```
Comment (corneliusweig) [inline plugins/kudo.yaml]: Minor nit:
You are using quite long lines, which will lead to strange line-wrapping on smaller terminals. Can you re-format to a width of 80 characters?

Comment (ahmetb) [reply plugins/kudo.yaml]: We can consider making this a lint error if necessary since we have a validator tool.

Comment (jbarrick-mesosphere) [reply plugins/kudo.yaml]: Updated :+1:

**Comment by jbarrick-mesosphere** [comment]
> If you have any advice you would want to show post-install, you can add a `caveats` field. For example, you could point to the documentation for writing kudo manifests and recommend to run `kubectl kudo -h` or so.

Added!

File: plugins/kudo.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  caveats: |
+    Example usage:
+      Install kafka:
+        kubectl kudo install kafka
+      List installed operator instances:
+        kubectl kudo get instances
+
+    See the documentation for more information: https://kudo.dev/docs/
+
+    Requires the KUDO controller to be installed:
```
Comment (ahmetb) [inline plugins/kudo.yaml]: you might wanna move requirements to the top.

Comment (jbarrick-mesosphere) [reply plugins/kudo.yaml]: Done :+1:

**Pull request merged**